draft-eggert-bcp45bis.md

```diff
@@ -164,12 +164,29 @@ Inappropriate postings include:
 
 # Security Considerations
 
-The IETF Chair, the IETF Executive Director, or a sergeant-at-arms {{SAA-SOP}}
-appointed by the Chair is empowered to restrict posting by a person, or of a
-thread, when the content is inappropriate and represents a pattern of abuse
-{{SAA-UPC}}. They are encouraged to take into account the overall nature of the
-postings by an individual and whether particular postings are an aberration or
-typical. Complaints regarding their decisions should be referred to the IAB.
+A sergeant-at-arms (SAA) appointed by the IETF Chair is empowered to
+restrict posting by a person, or of a thread, when the content is
+inappropriate and represents a pattern of abuse. They are encouraged to
+take into account the overall nature of the postings by an individual
+and whether particular postings are an aberration or typical.
+
+The sergeants-at-arms are intended to establish a self-moderation
+function on the community, by the community. The IETF Chair therefore
+should not appoint an SAA who is serving in a NomCom-appointed IETF
+leadership position. If an SAA is selected for such a position, they
+will step down as SAA.
+
+Apart from appointing SAAs, the IETF Chair should divorce themselves
+from the operation and management of the SAA team. This has been
+in practice for a while, and the SAA team has
+independently maintained definitions of abuse patterns {{SAA-UPC}} and
+operating procedures {{SAA-SOP}} for them. 
```

Comment:
Doesn't the IETF chair still serve as a last resort if there is a conflict that can't be resolved by the SAA?

Reply:
I think this would be useful if there is a disagreement between the SAA team members; the IETF chair could help resolve it. But I am not sure if we need to add text describing this...

Reply:
RFC3005 had:
>The IETF Chair, the IETF Executive Director, or a sergeant-at-arms
   appointed by the Chair is empowered to restrict posting by a person

I think the community has signalled that it doesn't want this power to remain with the IETF Chair, at least not directly. And I also think the changed role of the exec director makes them not very well suited for this.

Reply:
Yes, there were some/a few (?) people who raised that concern, however, at least one person also wanted an independent committee that is not appointed by the chair. So as long as the chair appoints the SAA, I don't see a big difference. I personally find it more important to make sure that everybody who serves as a SAA feels conformable with the position and for me that would also mean that they have someone they can approach and rely on in cause of problems, rather than having them taking full responsibility given this is a volunteer position.

Reply:
That does resonate with me as the SAA. How about - 

```suggestion
Apart from appointing SAAs, the IETF Chair should divorce themselves
from the day-to-day operation and management of the SAA team. This 
has been in practice for a while, and the SAA team has independently 
maintained definitions of abuse patterns {{SAA-UPC}} and operating 
procedures {{SAA-SOP}} for them. The SAA team should reach out to 
the IETF chair for any conflict resolution in a timely manner.
```

Reply:
Like it, but s/chair/Chair/ for consistency.